### DISCUSSION TITLE
Adding new institution

### DISCUSSION THREAD

**Discussion by Laertes2** [RosettaCommons/rosetta#124] [ROSIE]
Hello to all!
I am new in Rosie (and GitHub).
I need that Rosie recognizes mi institution as an academic user:
National Centre for Biotechnology (CNB-CSIC), Madrid Spain.
http://www.cnb.csic.es/index.php/en
(e-mail domain: @cnb.csic.es)
Thanks so much!

**Comment by lyskov** [comment]
Thank you for letting us know! This should be fixed now. Please feel free to re-open if you still can't loing.

**Comment by shgh776** [comment]
Hello to all!
I am new to Rosie (and GitHub).
I need Rosie to recognize my institution as an academic user:
https://en.pasteur.ac.ir/
email:info@pasteur.ac.ir
Thanks so much.

**Reply by roccomoretti** [reply]
See discussion #135
